src/Http/Session.php

```diff
@@ -545,7 +545,7 @@ private function sendCookie()
 		);
 		$this->response->setCookie(
 			'nette-browser', $_SESSION['__NF']['B'],
-			Response::BROWSER, $cookie['path'], $cookie['domain']
+			Response::BROWSER, $cookie['path'], $cookie['domain'], $cookie['secure']
```

Comment:
* Why not pass httponly option as well?
* This has been removed in https://github.com/nette/http/commit/0aed1a2bb65f42cb8a32f3909fd593538ac52f57, not sure why